airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import logging
```

Comment:
did 1 and 2 manually, since something wrong with (mine only?) installation of airbyte, can't finish connector setup process, check method just hangs forever.
1. connector consumes old config and syncs ok, same with state file
2. ui looks good in playbook from web-app

Reply:
WIP to test in human flow

Reply:
@sherifnada it is confirmed